package-lock.json

```diff
@@ -8,6 +8,8 @@
       "name": "fu-food",
       "version": "0.0.0",
       "dependencies": {
+        "sweetalert2": "^11.4.26",
```

Comment:
This file is package-lock.json, which contain specific version of package need to install, it might be diffrent with other devices.
You dont need to push changes on this file!
Please undo this file.